examples/some-verifier/README.md

```diff
@@ -65,3 +65,43 @@ Adds a new verification, taking JSON parameter examplified by the following obje
 ### PATCH `/verifications`
 
 Removes a verification. Takes a JSON parameter similar to as the corresponding `POST` endpoint, but will reject any proof made from anything else than 1 verifiable credential.
+
+
+## Docker image
+
+The docker image with the `some-verifier` can be built using the provided
+[`Dockerfile`](./scripts/build.Dockerfile).
+
+```console
+podman build     --build-arg build_image=rust:1.67-buster --build-arg frontend_build_image=node:16-slim --build-arg base_image=debian:buster -f examples/some-verifier/scripts/build.Dockerfile .
+```
+
+running from the **root** of the repository.
+
+This will produce a docker image with a binary `some-verifier` that is located in
+`/usr/local/bin`. That is meant to be the entrypoint of the image.
```

Comment:
This is a bit complex to do since some keys need to be supplied which involves mounting folders, etc.

In the interest of time I would like to merge this now and do another pass of documenting all of this once we have a deployment done.